sanskrit_parser/parser/datastructures.py

```diff
@@ -799,13 +826,8 @@ def get_purusha(self):
         ''' Get Node puruza '''
         return self.getNodeTagset().intersection(puruzas)
 
-    # FIXME: not needed?
-    def get_node_pos(self, node):
-        return self.path.index(node.pada.canonical())
```

Comment:
@kmadathil , this class does not have any member `path`, and this function would cause an error. So I removed it. Please cross-check.

Reply:
looks like it was obsolete but not removed.